spark/src/main/kotlin/com/adevinta/spark/components/toggles/CheckBox.md

```diff
@@ -47,6 +47,18 @@ CheckboxLabelled(
 ) { Text("CheckBox On") }
 ```
 
+### Styles
+
+The `CheckBox` and `CheckboxLabelled` accept 7 [ToggleIntent](ToggleIntent.kt)s:
```

Comment:
nit: trailing "s", is it supposed to be there ?

Reply:
Yes, so that it reads: 7 [ToggleIntent](ToggleIntent.kt)s